cfme/infrastructure/virtual_machines.py

```diff
@@ -311,6 +311,16 @@ class InfraVmSummaryView(VMDetailsEntities):
     datastore_usage = SummaryTable(title="Datastore Actual Usage Summary")
 
 
+class InfraVmContainerView(VMDetailsEntities):
+    basic_information = SummaryTable(locator='.//table[./thead/tr/th[normalize-space(.)='
```

Comment:
@kedark3 Can you just pass in the `title` of `Basic Information` without completely overwriting the BASELOC?

Reply:
Tried that. It doesn't work as the locator is different, hence adding ability to pass locator if desired. Giving more flexibility to user.

Reply:
John asked me offline if I can refactor the BASELOC to remove `align=left`, I am willing to mark that as TODO as it is not in the scope of PR. Let me know your thoughts.